### PR TITLE
Add use command to "rly keys"

### PR DESCRIPTION
**Pull request by vimystic**
Example : "rly keys use osmosis otherKeyName"
Note : if key does not exist or is already being used the command will return an error.

### CODE REVIEW

File: utils/configUtil.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
+package utils
```
Comment (agouin) [inline utils/configUtil.go]: config reading and writing is already implemented in rly. Here is an example: https://github.com/cosmos/relayer/blob/2fadd37236c7c3008037222a9350c7c4bcbf0eed/cmd/appstate.go#L224

File: cmd/keys.go [hunk 2]
```diff
@@ -55,6 +56,22 @@ func keysCmd(a *appState) *cobra.Command {
 	return cmd
 }
 
+func keysUseCmd(a *appState) *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:     "use chain_name key_name",
+		Aliases: []string{"a"},
+		Short:   "Use a key from the keychain associated with a particular chain. Look at ~/.relayer/keys/ibc-0/keyring-test ",
```
Comment (boojamya) [inline cmd/keys.go:64]: Nit: I don't think the "Look at ...path..." is necessary. 

My preference would be to either remove that sentence. OR maybe point them to the `rly keys list` command instead.

**Review by boojamya — APPROVED** [review]
Nice!
One minor nit.

File: cmd/keys.go (hunk 2 above)
Comment (vimystic) [reply cmd/keys.go:64]: Will do . I will make that update along with the read me file change .

**Pull request merged**